presto-product-tests/conf/product-tests-defaults.sh

```diff
@@ -1,13 +1,5 @@
 export DOCKER_IMAGES_VERSION=${DOCKER_IMAGES_VERSION:-28}
 
-if test -v HADOOP_BASE_IMAGE; then
-    test -v TESTS_HIVE_VERSION_MAJOR
-    test -v TESTS_HIVE_VERSION_MINOR
-else
-    test ! -v TESTS_HIVE_VERSION_MAJOR
-    test ! -v TESTS_HIVE_VERSION_MINOR
-
+if test ! -v HADOOP_BASE_IMAGE; then
     export HADOOP_BASE_IMAGE="prestodev/hdp2.6-hive"
-    export TESTS_HIVE_VERSION_MAJOR="1"
-    export TESTS_HIVE_VERSION_MINOR="2"
```

Comment:
This belongs to `Use product version obtained from Hive in Hive integration tests`
(the file is used in Hive integration tests only)